src/Common/src/Interop/Unix/System.Native/Interop.readdir.cs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Nit: The file is named Interop.readdir.cs, but all of the other files in this library are named to follow the naming used in the shim, i.e. Interop.ReadDir.cs.